src/parse.yy

```diff
@@ -569,7 +569,20 @@ intrinsic_name
     ;
 
 intrincall_expression
-    : intrinsic_name '(' argument_expression_list ')'
+    : intrinsic_name '(' ')'
+      {
+          std::string *name = $1;
+          name->erase(0, 1);
+          Symbol* sym = m->AddLLVMIntrinsicDecl(*name, nullptr, Union(@1,@3));
```

Comment:
I don't see any code in `AddLLVMIntrinsicDecl` verifying that `args` in not `nullptr` before accessing it. Could you add it?